etl/steps/data/garden/who/2023-04-03/flu_elderly.py

```diff
@@ -0,0 +1,54 @@
+"""Load a meadow dataset and create a garden dataset."""
+
+import pandas as pd
+from owid.catalog import Dataset, Table
+from structlog import get_logger
+
+from etl.data_helpers import geo
+from etl.helpers import PathFinder, create_dataset
+
+log = get_logger()
+
+# Get paths and naming conventions for current step.
+paths = PathFinder(__file__)
+
+
+def run(dest_dir: str) -> None:
+    log.info("flu_elderly.start")
+
+    #
+    # Load inputs.
+    #
+    # Load meadow dataset.
+    ds_meadow: Dataset = paths.load_dependency("flu_elderly")
+
+    # Read table from meadow dataset.
+    tb_meadow = ds_meadow["flu_elderly"]
+
+    # Create a dataframe with data from the table.
+    df = pd.DataFrame(tb_meadow).reset_index()
+
+    #
+    # Process data.
+    #
+    log.info("flu_elderly.harmonize_countries")
+    df = geo.harmonize_countries(
+        df=df, countries_file=paths.country_mapping_path, excluded_countries_file=paths.excluded_countries_path
```

Comment:
Optional: you can also delete `excluded_countries_file=paths.excluded_countries_path` together with the `flu_elderly.excluded_countries.json` file and the steps would still run